worlds/sc2/Starcraft2.kv

```diff
@@ -2,6 +2,12 @@
     scroll_type: ["content", "bars"]
     bar_width: dp(12)
     effect_cls: "ScrollEffect"
+    canvas.after:
+        Color:
+            rgba: (0.82, 0.2, 0, 1)
+        Line:
+            width: 1.5
+            rectangle: self.x+1 - 30*(not root.border_on), self.y+1 - 30*(not root.border_on), self.width-1 if root.border_on else 1, self.height-1 if root.border_on else 1
```

Comment:
It seems that .kv files don't allow spreading properties across multiple lines, and this calculation must be done here because only commands in .kv files are automatically re-executed when dependencies / the window size changes.

Note also that I implemented turning the border off by just making it 1x1, 30 pixels off screen to the bottom-left. It was the most reliable strategy I could come up with as:
* The default / background colour doesn't seem to be accessible
* Even if it was, it would draw background colour on top of buttons/the scrollbar
* Conditionally setting the colour to `None` crashes kivy
* Conditionally setting the colour to `()` causes kivy to
* get stuck in an infinite loop and hang while spewing out errors to stdout/stderr
* Setting the width to 0 causes kivy to crash (but with a different error message this time)

Suffice to say kivy gave me some grief.